{"date":"Sat Oct 20 2018 15:43:47 GMT+0530 (India Standard Time)","process":{"pid":1496,"uid":null,"gid":null,"cwd":"C:\\projects\\vidly","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v8.11.1","argv":["C:\\Program Files\\nodejs\\node.exe","C:\\projects\\vidly\\index"],"memoryUsage":{"rss":58224640,"heapTotal":40271872,"heapUsed":26837352,"external":18745536}},"os":{"loadavg":[0,0,0],"uptime":276398.2569223},"trace":[{"column":26,"file":"C:\\projects\\vidly\\index.js","function":null,"line":27,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":652,"method":"_compile","native":false},{"column":10,"file":"module.js","function":"Module._extensions..js","line":663,"method":".js","native":false},{"column":32,"file":"module.js","function":"Module.load","line":565,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":505,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":497,"method":"_load","native":false},{"column":10,"file":"module.js","function":"Module.runMain","line":693,"method":"runMain","native":false},{"column":16,"file":"bootstrap_node.js","function":"startup","line":188,"method":null,"native":false},{"column":3,"file":"bootstrap_node.js","function":null,"line":609,"method":null,"native":false}],"stack":["Error: something failed miserably","    at Object.<anonymous> (C:\\projects\\vidly\\index.js:27:26)","    at Module._compile (module.js:652:30)","    at Object.Module._extensions..js (module.js:663:10)","    at Module.load (module.js:565:32)","    at tryModuleLoad (module.js:505:12)","    at Function.Module._load (module.js:497:3)","    at Function.Module.runMain (module.js:693:10)","    at startup (bootstrap_node.js:188:16)","    at bootstrap_node.js:609:3"],"level":"error","message":"uncaughtException: something failed miserably","timestamp":"2018-10-20T10:13:47.749Z"}
{"date":"Sat Oct 20 2018 16:48:37 GMT+0530 (India Standard Time)","process":{"pid":3188,"uid":null,"gid":null,"cwd":"C:\\projects\\vidly","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v8.11.1","argv":["C:\\Program Files\\nodejs\\node.exe","C:\\projects\\vidly\\index"],"memoryUsage":{"rss":58118144,"heapTotal":40271872,"heapUsed":26245128,"external":18734235}},"os":{"loadavg":[0,0,0],"uptime":280287.743038},"trace":[{"column":11,"file":"C:\\projects\\vidly\\node_modules\\mongodb-core\\lib\\topologies\\server.js","function":null,"line":503,"method":null,"native":false},{"column":13,"file":"events.js","function":"emitOne","line":116,"method":null,"native":false},{"column":7,"file":"events.js","function":"Pool.emit","line":211,"method":"emit","native":false},{"column":12,"file":"C:\\projects\\vidly\\node_modules\\mongodb-core\\lib\\connection\\pool.js","function":null,"line":326,"method":null,"native":false},{"column":30,"file":"events.js","function":"Object.onceWrapper","line":317,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"emitTwo","line":126,"method":null,"native":false},{"column":7,"file":"events.js","function":"Connection.emit","line":214,"method":"emit","native":false},{"column":50,"file":"C:\\projects\\vidly\\node_modules\\mongodb-core\\lib\\connection\\connection.js","function":null,"line":245,"method":null,"native":false},{"column":30,"file":"events.js","function":"Object.onceWrapper","line":315,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"emitOne","line":116,"method":null,"native":false},{"column":7,"file":"events.js","function":"Socket.emit","line":211,"method":"emit","native":false},{"column":8,"file":"internal/streams/destroy.js","function":"emitErrorNT","line":64,"method":null,"native":false},{"column":11,"file":"internal/process/next_tick.js","function":"_combinedTickCallback","line":138,"method":null,"native":false},{"column":9,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":180,"method":"_tickCallback","native":false}],"stack":["MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","    at Pool.<anonymous> (C:\\projects\\vidly\\node_modules\\mongodb-core\\lib\\topologies\\server.js:503:11)","    at emitOne (events.js:116:13)","    at Pool.emit (events.js:211:7)","    at Connection.<anonymous> (C:\\projects\\vidly\\node_modules\\mongodb-core\\lib\\connection\\pool.js:326:12)","    at Object.onceWrapper (events.js:317:30)","    at emitTwo (events.js:126:13)","    at Connection.emit (events.js:214:7)","    at Socket.<anonymous> (C:\\projects\\vidly\\node_modules\\mongodb-core\\lib\\connection\\connection.js:245:50)","    at Object.onceWrapper (events.js:315:30)","    at emitOne (events.js:116:13)","    at Socket.emit (events.js:211:7)","    at emitErrorNT (internal/streams/destroy.js:64:8)","    at _combinedTickCallback (internal/process/next_tick.js:138:11)","    at process._tickCallback (internal/process/next_tick.js:180:9)"],"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","timestamp":"2018-10-20T11:18:37.238Z"}
{"date":"Wed Dec 12 2018 10:07:10 GMT+0530 (India Standard Time)","process":{"pid":6204,"uid":null,"gid":null,"cwd":"C:\\projects\\vidlywithtests","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v8.11.1","argv":["C:\\Program Files\\nodejs\\node.exe","C:\\projects\\vidlywithtests\\index.js","index.js"],"memoryUsage":{"rss":53714944,"heapTotal":37085184,"heapUsed":20564264,"external":18889280}},"os":{"loadavg":[0,0,0],"uptime":392921.3427707},"trace":[{"column":15,"file":"C:\\projects\\vidlywithtests\\startup\\config.js","function":"module.exports","line":5,"method":"exports","native":false},{"column":28,"file":"C:\\projects\\vidlywithtests\\index.js","function":null,"line":8,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":652,"method":"_compile","native":false},{"column":10,"file":"module.js","function":"Module._extensions..js","line":663,"method":".js","native":false},{"column":32,"file":"module.js","function":"Module.load","line":565,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":505,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":497,"method":"_load","native":false},{"column":10,"file":"module.js","function":"Module.runMain","line":693,"method":"runMain","native":false},{"column":16,"file":"bootstrap_node.js","function":"startup","line":188,"method":null,"native":false},{"column":3,"file":"bootstrap_node.js","function":null,"line":609,"method":null,"native":false}],"stack":["Error: FATAL ERROR: jwtPrivateKey not defined","    at module.exports (C:\\projects\\vidlywithtests\\startup\\config.js:5:15)","    at Object.<anonymous> (C:\\projects\\vidlywithtests\\index.js:8:28)","    at Module._compile (module.js:652:30)","    at Object.Module._extensions..js (module.js:663:10)","    at Module.load (module.js:565:32)","    at tryModuleLoad (module.js:505:12)","    at Function.Module._load (module.js:497:3)","    at Function.Module.runMain (module.js:693:10)","    at startup (bootstrap_node.js:188:16)","    at bootstrap_node.js:609:3"],"level":"error","message":"uncaughtException: FATAL ERROR: jwtPrivateKey not defined","timestamp":"2018-12-12T04:37:10.849Z"}
{"date":"Wed Dec 12 2018 10:08:15 GMT+0530 (India Standard Time)","process":{"pid":11952,"uid":null,"gid":null,"cwd":"C:\\projects\\vidlywithtests","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v8.11.1","argv":["C:\\Program Files\\nodejs\\node.exe","C:\\projects\\vidlywithtests\\index.js","index.js"],"memoryUsage":{"rss":55803904,"heapTotal":36601856,"heapUsed":26169056,"external":18872896}},"os":{"loadavg":[0,0,0],"uptime":392985.9950059},"trace":[{"column":15,"file":"C:\\projects\\vidlywithtests\\startup\\config.js","function":"module.exports","line":5,"method":"exports","native":false},{"column":28,"file":"C:\\projects\\vidlywithtests\\index.js","function":null,"line":8,"method":null,"native":false},{"column":30,"file":"module.js","function":"Module._compile","line":652,"method":"_compile","native":false},{"column":10,"file":"module.js","function":"Module._extensions..js","line":663,"method":".js","native":false},{"column":32,"file":"module.js","function":"Module.load","line":565,"method":"load","native":false},{"column":12,"file":"module.js","function":"tryModuleLoad","line":505,"method":null,"native":false},{"column":3,"file":"module.js","function":"Module._load","line":497,"method":"_load","native":false},{"column":10,"file":"module.js","function":"Module.runMain","line":693,"method":"runMain","native":false},{"column":16,"file":"bootstrap_node.js","function":"startup","line":188,"method":null,"native":false},{"column":3,"file":"bootstrap_node.js","function":null,"line":609,"method":null,"native":false}],"stack":["Error: FATAL ERROR: jwtPrivateKey not defined","    at module.exports (C:\\projects\\vidlywithtests\\startup\\config.js:5:15)","    at Object.<anonymous> (C:\\projects\\vidlywithtests\\index.js:8:28)","    at Module._compile (module.js:652:30)","    at Object.Module._extensions..js (module.js:663:10)","    at Module.load (module.js:565:32)","    at tryModuleLoad (module.js:505:12)","    at Function.Module._load (module.js:497:3)","    at Function.Module.runMain (module.js:693:10)","    at startup (bootstrap_node.js:188:16)","    at bootstrap_node.js:609:3"],"level":"error","message":"uncaughtException: FATAL ERROR: jwtPrivateKey not defined","timestamp":"2018-12-12T04:38:15.502Z"}
{"date":"Wed Dec 12 2018 14:53:01 GMT+0530 (India Standard Time)","process":{"pid":10808,"uid":null,"gid":null,"cwd":"C:\\projects\\vidlywithtests","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v8.11.1","argv":["C:\\Program Files\\nodejs\\node.exe","C:\\projects\\vidlywithtests\\index"],"memoryUsage":{"rss":53006336,"heapTotal":37126144,"heapUsed":23214096,"external":17984277}},"os":{"loadavg":[0,0,0],"uptime":14459.9066018},"trace":[{"column":11,"file":"C:\\projects\\vidlywithtests\\node_modules\\mongodb-core\\lib\\topologies\\server.js","function":null,"line":503,"method":null,"native":false},{"column":13,"file":"events.js","function":"emitOne","line":116,"method":null,"native":false},{"column":7,"file":"events.js","function":"Pool.emit","line":211,"method":"emit","native":false},{"column":12,"file":"C:\\projects\\vidlywithtests\\node_modules\\mongodb-core\\lib\\connection\\pool.js","function":null,"line":326,"method":null,"native":false},{"column":30,"file":"events.js","function":"Object.onceWrapper","line":317,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"emitTwo","line":126,"method":null,"native":false},{"column":7,"file":"events.js","function":"Connection.emit","line":214,"method":"emit","native":false},{"column":50,"file":"C:\\projects\\vidlywithtests\\node_modules\\mongodb-core\\lib\\connection\\connection.js","function":null,"line":245,"method":null,"native":false},{"column":30,"file":"events.js","function":"Object.onceWrapper","line":315,"method":"onceWrapper","native":false},{"column":13,"file":"events.js","function":"emitOne","line":116,"method":null,"native":false},{"column":7,"file":"events.js","function":"Socket.emit","line":211,"method":"emit","native":false},{"column":8,"file":"internal/streams/destroy.js","function":"emitErrorNT","line":64,"method":null,"native":false},{"column":11,"file":"internal/process/next_tick.js","function":"_combinedTickCallback","line":138,"method":null,"native":false},{"column":9,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":180,"method":"_tickCallback","native":false}],"stack":["MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","    at Pool.<anonymous> (C:\\projects\\vidlywithtests\\node_modules\\mongodb-core\\lib\\topologies\\server.js:503:11)","    at emitOne (events.js:116:13)","    at Pool.emit (events.js:211:7)","    at Connection.<anonymous> (C:\\projects\\vidlywithtests\\node_modules\\mongodb-core\\lib\\connection\\pool.js:326:12)","    at Object.onceWrapper (events.js:317:30)","    at emitTwo (events.js:126:13)","    at Connection.emit (events.js:214:7)","    at Socket.<anonymous> (C:\\projects\\vidlywithtests\\node_modules\\mongodb-core\\lib\\connection\\connection.js:245:50)","    at Object.onceWrapper (events.js:315:30)","    at emitOne (events.js:116:13)","    at Socket.emit (events.js:211:7)","    at emitErrorNT (internal/streams/destroy.js:64:8)","    at _combinedTickCallback (internal/process/next_tick.js:138:11)","    at process._tickCallback (internal/process/next_tick.js:180:9)"],"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","timestamp":"2018-12-12T09:23:01.410Z"}
